deploy/000_deploy_postage.ts

```diff
@@ -1,29 +1,38 @@
-import { HardhatRuntimeEnvironment } from 'hardhat/types';
 import { DeployFunction } from 'hardhat-deploy/types';
+import { networkConfig, developmentChains, deployedBzzData } from '../helper-hardhat-config';
 
-const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
-  const { deployments, getNamedAccounts } = hre;
-  const { deploy, execute, read } = deployments;
+const func: DeployFunction = async function ({ deployments, getNamedAccounts, network, ethers }) {
+  const { deploy, log } = deployments;
+  const { deployer } = await getNamedAccounts();
```

Comment:
it is a bit ugly here the `deployer` can be retrieved from `getNamedAccounts`...

Reply:
Not sure what is suggestion here. Can you reiterate?

Reply:
at first glance I thought that is a deployer instance and actually I wanted to delete the comment after.